doc/v2.1-joss.md

```diff
@@ -0,0 +1,107 @@
+          - 6
+    - name: Shannon Joslin
+      orcid:
+      affiliation: 2
+    - name: Nicole Kingsley
+      orcid:
+      affiliation: 2
+    - name: Kevin D. Murray
+      orcid:
+      affiliation: 7
+    - name: Camille Scott
+      orcid:
+      affiliation:
+          - 1
+          - 6
+    - name: Sascha Steinbiss
+      orcid:
+      affiliation: 8
+    - name: Cait Sydney
+      orcid:
+      affiliation: 9
+    - name: C. Titus Brown
+      orcid:
+      affiliation:
+          - 1
+          - 10
+affiliations:
+    - name: Lab for Data Intensive Biology; School of Veterinary Medicine; University of California, Davis
+      index: 1
+    - name: Integrative Genetics and Genomics Graduate Group; University of California, Davis
+      index: 2
+    - name: Molecular, Cellular, and Integrative Physiology Graduate Group; University of California, Davis
+      index: 3
+    - name: Common Workflow Language Project
+      index: 4
+    - name: Wild Tree Tech
+      index: 5
+    - name: Computer Science Graduate Group; University of California
+      index: 6
+    - name: ARC Centre of Excellence in Plant Energy Biology; Australian National University
+      index: 7
+    - name: Parasite Genomics, Wellcome Trust Sanger Institute
+      index: 8
+    - name: Google, Inc.
+      index: 9
+    - name: Department of Population Health and Reproduction; School of Veterinary Medicine; University of California, Davis
+      index: 10
+date: 2017-05-23
+---
+
+# Summary
+
+The khmer software is a set of command-line tools built around a Python library designed for analysis of large DNA sequence collections.
+Functionality in khmer has primarily been motivated by scaling issues with (meta)genome and (meta)transcriptome assembly (Crusoe *et al.*, 2015).
+khmer provides convenient access to several k-mer based operations on DNA sequence collections, such as abundance filtering, error trimming, and assembly graph partitioning.
+All of these operations utilize khmer's implementation of two primary data structures, the Bloom filter and the Count-Min Sketch, for efficient probabalistic storage of k-mer presence or k-mer abundance, respectively (Pell *et al.*, 2012; Zhang *et al.*, 2014).
+
+Release version 2.1 of the khmer software includes several new features that extend its utility to a wider set of sequence processing and analysis problems.
+These include the following:
+support for variable-coverage trimming of sequence reads;
+support for k > 32 using the non-reversible hash function MurmurHash3 (Appleby, 2010);
+a new optional Count-Min Sketch implementation providing increased storage efficiency;
+support for assembly directly from a k-mer graph;
+a script for computing a compact de Bruijn graph from a k-mer graph;
+and several examples of khmer's Python and C++ APIs for those interested in using and extending the library.
+
+# References
+
+**Appleby A** (2010) SMHasher. *GitHub*, https://github.com/aappleby/smhasher, accessed May 22, 2017.
```

Comment:
The author never "published" MurmurHash3, and has placed it in the public domain with no date. I used the year of the first git commit in the repository as the publication date. Kosher?